Vizzuality/trase
# Feat/country profile links

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/171568692

## Description

Profile links for country profiles, reusing the ones we had previously only parsing is a bit different for the links. Also fixed a small issue where we try to update state on a unmounted component. `banner.component` 

## Testing instructions

Test both export/import countries, make sure links correspond to the correct page.
